### PR TITLE
Sarka/storybook migration b5

### PR DESCRIPTION
**Pull request by sarkaaa**
# This Pull Request meets the following criteria:

### For new components:

- [ ] Unit and visual regression tests have been added/adjusted for my new feature
- [ ] New Components are registered in index.js of my project
- [ ] New Components have `d.ts` files and are exported in `index.d.ts`

Closes https://kiwicom.atlassian.net/browse/FEPLT-2076

### CODE REVIEW

File: packages/orbit-components/src/OrbitProvider/OrbitProvider.stories.tsx [hunk 1]
```diff
@@ -1,50 +1,68 @@
...
-  customTheme: { black: "#000" },
 };
 
-export default {
-  title: "OrbitProvider",
+type Story = StoryObj<OrbitProviderPropsAndCustomArgs>;
+
+export default meta;
+
+export const Default: Story = {
```
Comment (sarkaaa) [inline packages/orbit-components/src/OrbitProvider/OrbitProvider.stories.tsx:59]: I haven't found any better solution how to make this story simpler/smoother. If you have any suggestions, feel free to let me know. ❤️

File: packages/orbit-components/src/Radio/Radio.stories.tsx [hunk 2]
```diff
@@ -1,202 +1,146 @@
...
-export const WithError = ({ label, hasError, checked }) => {
-  return <Radio label={label} hasError={hasError} checked={checked} onChange={action("changed")} />;
+  args: {
+    info: undefined,
+  },
 };
 
-WithError.story = {
-  name: "With error",
+export const WithError = {
```
[the remark this answers is not in the document]
Comment (sarkaaa) [reply packages/orbit-components/src/Radio/Radio.stories.tsx:53]: omg, I don't understand how I could skip it. 🙈

File: packages/orbit-components/src/Pagination/Pagination.stories.tsx [hunk 3]
```diff
@@ -1,215 +1,94 @@
...
-  labelPrev: "prev",
-  labelNext: "next",
-  labelProgress: "of",
-};
+  args: {
+    selectedPage: 1,
+    pageCount: 99,
+    size: SIZES.NORMAL,
+    hideLabels: false,
+  },
```
[the remark this answers is not in the document]
Comment (sarkaaa) [reply packages/orbit-components/src/Pagination/Pagination.stories.tsx:64]: yes, I didn't include `id` in Pagination as it's not used anywhere across the component (it's specified in `Common.Globals`, but that's all).

File: packages/orbit-components/src/Radio/Radio.stories.tsx [hunk 4]
```diff
@@ -1,202 +1,146 @@
...
-export const WithHelp = ({ label, value, info }) => {
-  return <Radio label={label} value={value} info={info} onChange={action("changed")} />;
-};
+export default meta;
 
-WithHelp.story = {
-  name: "With help",
+type Story = StoryObj<typeof Radio>;
 
+export const Default: Story = {
```
[the remark this answers is not in the document]
Comment (sarkaaa) [reply packages/orbit-components/src/Radio/Radio.stories.tsx:40]: I kept those values as they're commonly related to Radio input element. Any of the props are not mandatory (label, onClick), but I kept them anyway.

File: packages/orbit-components/src/SegmentedSwitch/SegmentedSwitch.stories.tsx [hunk 5]
```diff
@@ -1,78 +1,64 @@
...
+    error: "Chuck Norris makes onions cry.",
+    maxWidth: "20%",
+    showTooltip: false,
+    options: [
+      { label: "Male", value: "Male" },
+      { label: "Female", value: "Female" },
+    ],
+    onChange: action("onChange"),
+    onFocus: action("onFocus"),
+    spaceAfter: SPACINGS_AFTER.NORMAL,
```
[the remark this answers is not in the document]
Comment (sarkaaa) [reply packages/orbit-components/src/SegmentedSwitch/SegmentedSwitch.stories.tsx:33]: Same as for the story/component above. `id` is not used anywhere within the component.

File: packages/orbit-components/src/Select/Select.stories.tsx [hunk 6]
```diff
@@ -16,237 +17,208 @@ const objectOptions = [
...
-    error={<div>You need to select some value.</div>}
-    onChange={action("onChange")}
-  />
-);
-
-WithErrorMessage.story = {
-  name: "With error message",
+export const WithHelpMessage = {
+  args: {
+    help: "Most common choice is Booking cancellation",
```
[the remark this answers is not in the document]
Comment (sarkaaa) [reply packages/orbit-components/src/Select/Select.stories.tsx:151]: ok, could be. 👍

File: packages/orbit-components/src/Select/Select.stories.tsx [hunk 7]
```diff
@@ -16,237 +17,208 @@ const objectOptions = [
...
+    label: "Select",
+    placeholder: "Select value from list",
+    name: "name",
+    customValueText: "",
+    value: "value",
+    spaceAfter: SPACINGS_AFTER.SMALL,
+    id: "select-id",
+    required: false,
+    width: "50%",
+    prefix: "Prefix",
```
[the remark this answers is not in the document]
Comment (sarkaaa) [reply packages/orbit-components/src/Select/Select.stories.tsx:186]: Do we know that `prefix` will always be an icon and not text? I mean, it makes sense for me and I don't remember if I noticed there was anything different than icon/flag, but I'm just thinking about all possible use cases.

File: packages/orbit-components/src/Popover/Popover.stories.tsx [hunk 8]
```diff
@@ -523,149 +457,59 @@ export const Playground = ({
...
+    fixed: true,
+    lockScrolling: true,
+    noFlip: false,
+    renderTimeout: 0,
+    allowOverflow: false,
+    noPadding: false,
+    width: "350px",
+    maxHeight: "",
+    overlapped: false,
+    footerActions: true,
```
[the remark this answers is not in the document]
Comment (sarkaaa) [reply packages/orbit-components/src/Popover/Popover.stories.tsx:488]: nono, I just forgot to add `...Placement.args`. 🙈

File: packages/orbit-components/src/Select/Select.stories.tsx (hunk 7 above)
[the remark this answers is not in the document]
Comment (domihustinova) [reply packages/orbit-components/src/Select/Select.stories.tsx]: If someone would want to put there text, it makes more sense to me to use `inlineLabel` for that. In the documentation there is actually a recommendation to use an icon/flag there 👀 https://orbit.kiwi/components/input/select/react/#functional-specs

[the remark this answers is not in the document]
Comment (sarkaaa) [reply packages/orbit-components/src/Select/Select.stories.tsx]: Changed. 👍

File: packages/orbit-components/src/Select/Select.stories.tsx [hunk 9]
```diff
@@ -16,237 +17,230 @@ const objectOptions = [
...
+    info: "Select component with country flag icon as a prefix. Check Orbit.Kiwi for more detailed guidelines.",
+  },
 };
 
-WithPlaceholder.story = {
-  name: "With placeholder",
+export const WithLongLabel: Story = {
+  args: {
+    label: "Select box (very long label)",
+  },
```
Comment (domihustinova) [inline packages/orbit-components/src/Select/Select.stories.tsx:125]: Why did you remove `inlineLabel`? I think it made sense to be here - especially if you are saying in the `info` that `long label truncate automatically when inline`.

Comment (domihustinova) [reply packages/orbit-components/src/Select/Select.stories.tsx:125]: Why did you remove `inlineLabel`? I think it made sense to be here - especially if you are saying in the `info` that `long label truncate automatically when inline`.

File: packages/orbit-components/src/Select/Select.stories.tsx [hunk 10]
```diff
@@ -16,237 +17,230 @@ const objectOptions = [
...
-  spaceAfter: {
-    options: Object.values(SPACINGS_AFTER),
-    control: {
-      type: "select",
+    return <Select {...args} prefix={<Icon />} />;
+  },
+
+  args: {
+    ...WithPrefixAndLabel.args,
+    label: "Select",
```
Comment (domihustinova) [inline packages/orbit-components/src/Select/Select.stories.tsx:186]: `label` is already specified in `WithPrefixAndLabel.args` so it's unnecessary here, no?

File: packages/orbit-components/src/Select/Select.stories.tsx [hunk 11]
```diff
@@ -16,237 +17,230 @@ const objectOptions = [
...
+    value: undefined,
+    disabled: false,
+    error: "",
+    help: "",
+    name: "name",
+    width: "50%",
+    tabIndex: 0,
+    required: false,
+    helpClosable: true,
+    prefix: "Prefix",
```
Comment (domihustinova) [inline packages/orbit-components/src/Select/Select.stories.tsx:198]: Now when you added `icon` here, `prefix` doesn't do anything, no?

File: packages/orbit-components/src/Select/Select.stories.tsx (hunk 9 above)
Comment (sarkaaa) [reply packages/orbit-components/src/Select/Select.stories.tsx:125]: I think it was there during some of the "draft" version, but it disappeared later and I didn't notice that. GP.

**Review by domihustinova — APPROVED** [review]
